scanpipe/tests/test_models.py

```diff
@@ -1020,3 +1022,60 @@ def test_scanpipe_discovered_package_model_create_from_data(self):
         self.assertEqual(expected_message, error.message)
         self.assertEqual(bad_data["version"], error.details["version"])
         self.assertIn("in save", error.traceback)
+
+
+class ScanPipeWalkTest(BaseScanPipeModelsTest, TestCase):
+    data_location = Path(__file__).parent / "data"
+    fixtures = [data_location / "asgiref-3.3.0_walk_test_fixtures.json"]
```

Comment:
@tdruez To avoid using `# noqa: E501` at the  end of the long path strings, I shortened the path strings in `asgiref-3.3.0_fixtures.json` and placed the `CodebaseResource` walk test in its own class.